provisioning/README.md

```diff
@@ -23,8 +23,32 @@ You'll need to create you openshift router and registry (see https://github.com/
 
 ## Fully Automated Environment Provisioning
 
-See sub-directories for environment specific provisioning scripts and instructions.
+We now support full end-to-end Environment Provisioning.
 
 Current implementations:
 
  - Openstack
+
+### Instructions
+
+1. [Configure client tools](provisioning/openstack/README.md)
+2. Clone this repo.
+3. Run provisioning script:
+```bash
+# Run osc-provision with no options to show Usage output and options
+$ ./ose-utils/provisioning/osc-provision
+Missing argument: --num-nodes <integer>
+
+Usage: ./provisioning/osc-provision --num-nodes=<integer> [options]
+
+Options:
+--openshift-domain=<domain>   : Base domain name for your OpenShift environment (default: ose.example.com)
+--cloudapp-domain=<domain>    : Wildcard domain for your applications (default: *.ose.example.com)
```

Comment:
Not this document's issue, but looks like the "default" is wrong as I believe it's *.apps.ose.example.com)